tests/modules/individuals/resources/test_individual_names.py

```diff
@@ -5,11 +5,11 @@
 from tests.modules.individuals.resources import utils as individual_utils
 from tests.modules.sightings.resources import utils as sighting_utils
 
+from app.modules.encounters.models import Encounter
```

Comment:
I think that having models included at the top level in test files has caused problems in the past with circular inheritance. 
I don't have any evidence to back this up but I have a vague memory that it's caused us problems in the past so I avoid it now.

Reply:
interesting!  noted.   <fear>

Reply:
Yes, this was the case for a while but Karen did some work to fix the behavior.